### PR TITLE
SAK-40378: Sakai 12.x spanish translation update for 12.3 release (after last merges)

### PR DESCRIPTION
**Pull request by juanjmerono**
…ter last merges)

### CODE REVIEW

**Comment by juanjmerono** [comment]
@jesusmmp Can you take a look before merge it?

File: content-review/impl/turnitin-oc/src/main/resources/turnitin_es.properties [hunk 1]
```diff
@@ -0,0 +1,18 @@
+file.type.doc=Word 97-2003
```
Comment (jonespm) [inline content-review/impl/turnitin-oc/src/main/resources/turnitin_es.properties]: Are these supposed to be translated or are they supposed to be static? Are these called the same thing in Spanish?

Comment (jesusmmp) [reply content-review/impl/turnitin-oc/src/main/resources/turnitin_es.properties]: In Spanish is the same thing.

**Comment by jesusmmp** [comment]
I think this:
-sendFeedback.overwrite=Sobreescribir comentarios actuales (NOTA\: Si no se marca esta opci\u00f3n, los comentarios se a\u00f1aden a continuaci\u00f3n de los comentarios existentes).

is better than the new one. I'll talk tomorrow with @juanjmerono 

The other translations are OK. Thanks

**Pull request merged**